src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -617,6 +627,22 @@ type VehicleRentalStation implements Node & PlaceInterface {
     operative: Boolean
 }
 
+type RentalVehicleEntityCounts {
+    """The total number of entities (e.g. vehicles, spaces)."""
+    total: Int!
```

Comment:
In GBFS they are mandatory. Are you thinking of your own custom importer for Vilkuu?

Reply:
I'm not sure about this. The 2.3 documentation says for num_docks_available "REQUIRED except for stations that have unlimited docking capacity (for example, valet stations) ". We got rid of the Vilkku updater as they now provide gbfs but it still has similar problems and it can't be the only network in the world where they are not able to provide real numbers for everything.

Reply:
Ok, lets have a chat on the call today.